p2panda-rs/src/secret_group/mod.rs

```diff
@@ -0,0 +1,64 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+//! Create and moderate large secret groups to maintain key material for secure messaging.
+//!
+//! p2panda uses the [MLS] (Messaging Layer Security) protocol for group key negotiation to
+//! establish secrets in a large group of users for "Sender Ratchet Secrets" or "Long Term
```

Comment:
I would personally format the special terms using _italics_ instead of ["quotes"](https://mads.de/awkward_anfuehrungszeichen-zeigt-die-grosse-wirkung-kleiner-satzzeichen/) but ¯\\\_(ツ)\_\/¯

Reply:
I removed "large", not sure if thats important at this point?